src/cli/utils/listing.rs

```diff
@@ -28,16 +30,48 @@ pub async fn find_lune_scripts(in_home_dir: bool) -> Result<Vec<(String, String)
                 let meta = entry.metadata().await?;
                 if meta.is_file() {
                     let contents = fs::read(entry.path()).await?;
+                    files.push((entry, meta, contents));
+                } else if meta.is_dir() {
+                    let entry_path_os = entry.path();
+
+                    let Some(dir_path) = entry_path_os.to_str() else {
+                        bail!("Failed to cast path to string slice.")
+                    };
+
+                    let init_file_path = match (
+                        discover_script_path(dir_path, in_home_dir),
+                        discover_script_path(dir_path, in_home_dir),
+                    ) {
```

Comment:
I'm a bit confused by this match here, why are we calling `discover_script_path` twice with the same args? 🤔

Reply:
My bad, initially I wasn't aware of previous match for `lune` and `.lune`, so I was going to check for both.

Fixed in https://github.com/0x5eal/lune-list-init/commit/299a80a2e8b65a4ec739e3f125f9f2e318803b35.